jscomp/test/ImportAttributes.res

```diff
@@ -0,0 +1,6 @@
+@@config({flags: ["-bs-package-output", "es6:jscomp/test:.mjs"]})
+
+@module({from: "./myJson.json", with: {type_: "json"}})
+external myJson: Js.Json.t = "default"
+
```

Comment:
```suggestion
@module({from: "./myJson.json", with: {type_: "json"}})
external myJson: Js.Json.t = "default"

@module({from: "./myModule", with: {foo: "foo", "bar-baz": "only-string-allowed"}})
external moreAttributes: unit = "default"

@module({with: {type_: "css"}})
external styles: Styles.t = "./myModule.css"
```

Reply:
Yeah it doesn't handle invalid identifiers yet, but that would be easy to handle by just outputting quotes around the keys. That should be fine, right?

You'd have to escape the identifiers BTW, it needs to be a record in the AST.

Reply:
@cometkim I updated the code to handle this case, and added an example to the test output. Let me know what you think.